### PR TITLE
Feature/liayoo/init

### PR DESCRIPTION
**Pull request by liayoo**
Added some skeleton code and tests.
Note that the API has not been tested yet (will be tested once the server & domain are set up).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "repository": "git@github.com:ainft-team/ainft-js.git",
+  "keywords": [
+    "ainetwork",
+    "ain",
+    "TypeScript",
+    "AINFT",
+    "NFT",
+    "SDK"
+  ],
+  "author": "Crypto X Team",
```
Comment (platfowner) [inline package.json:25]: minor) as it's a public repo, what about something like 'AINFT Team'?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: package.json (hunk 1 above)
Comment (liayoo) [reply package.json]: Right, it makes sense to use the same name as the org. Thanks!

**Comment by liayoo** [comment]
Thanks for the quick review. I'll merge this PR after testing the API!

**Pull request merged**